### PR TITLE
Create prometheus.md

### PR DESCRIPTION
**Pull request by spacelift-ryan**
# Description of the change

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- You have updated the navigation files correctly:
    - [ ] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [x] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/integrations/observability/prometheus.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Prometheus Integration
+
+The Prometheus exporter allows you to monitor various metrics about your Spacelift account over time. You can then use tools like Grafana to visualize those changes and Alertmanager to take actions based on account metrics. Several metrics are available, and you can find the complete list of available metrics here.
```
Comment (adamconnelly) [inline docs/integrations/observability/prometheus.md:3]:
```suggestion
The Prometheus exporter allows you to monitor various metrics about your Spacelift account over time. You can then use tools like Grafana to visualize those changes and Alertmanager to take actions based on account metrics. Several metrics are available, and you can find the complete list of available metrics [here](https://github.com/spacelift-io/prometheus-exporter#available-metrics). 
```

File: docs/integrations/observability/prometheus.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Prometheus Integration
+
+The Prometheus exporter allows you to monitor various metrics about your Spacelift account over time. You can then use tools like Grafana to visualize those changes and Alertmanager to take actions based on account metrics. Several metrics are available, and you can find the complete list of available metrics here. 
+
+How Does It Work?
```
Comment (adamconnelly) [inline docs/integrations/observability/prometheus.md:5]:
```suggestion
## How Does It Work?

```

File: docs/integrations/observability/prometheus.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+* Getting API Credentials
+
+* Installing the Prometheus Exporter
+
+* Viewing your Metrics
+
+* Alerting
+
+* Autoscaling
```
Comment (adamconnelly) [inline docs/integrations/observability/prometheus.md:22]: Is the plan to just link back to the GitHub README, or are you wanting to actually include this content in the docs? If you're just going to link, I'd remove these changes.

**Review by adamconnelly — COMMENTED** [review]
It looks like the changes to add the new file to the nav are missing. You need to update the following two places to add the menu items to the docs:

- https://github.com/spacelift-io/user-documentation/blob/b60af42fd11406ac679636cbf07ce6b1cb16b2db/nav.yaml#L135
- https://github.com/spacelift-io/user-documentation/blob/b60af42fd11406ac679636cbf07ce6b1cb16b2db/nav.self-hosted.yaml#L118

The first is for SaaS - the second is for Self-Hosted. I think it makes sense in both because it should work fine with Self-Hosted.

File: docs/integrations/observability/prometheus.md (hunk 3 above)
Comment (spacelift-ryan) [reply docs/integrations/observability/prometheus.md:22]: I want to include this content in the docs. I did it to just get it started so the team can help finish it.

**Comment by spacelift-ryan** [comment]
@adamconnelly Let me know if my updates resolved the issue to merge.

**Comment by adamconnelly** [comment]
> @adamconnelly Let me know if my updates resolved the issue to merge.

The nav changes look good, thanks.

I think we still need to figure out what's happening with this comment though: https://github.com/spacelift-io/user-documentation/pull/348#discussion_r1352876756.

If you want to merge as-is, I think we should remove those points, otherwise the docs page is going to look really weird. What's the plan, are you wanting to merge this as a placeholder page, and then someone else picks it up to finish, or are you wanting those sections filled out before the change is merged and goes live?

**Comment by spacelift-ryan** [comment]
@adamconnelly I removed the bulleted points for now.

File: docs/integrations/observability/prometheus.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Prometheus Integration
```
Comment (adamconnelly) [inline docs/integrations/observability/prometheus.md:1]:
```suggestion
---
description: Exporting Spacelift data to Prometheus
---

# Prometheus integration
```

File: docs/integrations/observability/prometheus.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+The Prometheus exporter allows you to monitor various metrics about your Spacelift account over time. You can then use tools like Grafana to visualize those changes and Alertmanager to take actions based on account metrics. Several metrics are available, and you can find the complete list of available metrics [here](https://github.com/spacelift-io/prometheus-exporter#available-metrics).
+
+## How Does It Work?
```
Comment (adamconnelly) [inline docs/integrations/observability/prometheus.md:5]:
```suggestion
## How it works
```

File: docs/integrations/observability/prometheus.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+description: Exporting Spacelift data to Prometheus
+---
+
+# Prometheus integration
+description: Exporting Spacelift data to Prometheus
```
Comment (adamconnelly) [inline docs/integrations/observability/prometheus.md:6]:
```suggestion
```

File: docs/integrations/observability/prometheus.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+---
```
Comment (adamconnelly) [inline docs/integrations/observability/prometheus.md:7]:
```suggestion
```

File: docs/integrations/observability/prometheus.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+description: Exporting Spacelift data to Prometheus
+---
+
+# Prometheus integration
```
Comment (adamconnelly) [inline docs/integrations/observability/prometheus.md:5]:
```suggestion
```

**Review by adamconnelly — DISMISSED** [review]

**Review by adamconnelly — APPROVED** [review]

**Pull request merged**